src/components/Checkout/OrderSummary.js

```diff
@@ -73,18 +83,43 @@ const OrderSummary = ({ checkout }) => {
 
     stripe.retrievePaymentIntent(clientSecret).then(({ paymentIntent }) => {
       switch (paymentIntent.status) {
-        case 'succeeded':
-          setMessage('Payment succeeded!');
+        case 'succeeded': {
+          toast({
+            title: 'Payment succeeded.',
+            status: 'success',
+            duration: 9000,
+            isClosable: true,
+          });
           break;
-        case 'processing':
-          setMessage('Your payment is processing.');
+        }
+
+        case 'processing': {
+          toast({
+            title: 'Payment processing.',
+            status: 'warning',
```

Comment:
maybe change status to `info` instead of warning

Reply:
done